server/ai_http.go

```diff
@@ -97,7 +97,14 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		remoteAddr := getRemoteAddr(r)
 		ctx := clog.AddVal(r.Context(), clog.ClientIP, remoteAddr)
-		requestID := string(core.RandomManifestID())
+		streamID := r.Header.Get("streamID")
+		requestID := r.Header.Get("requestID")
```

Comment:
Not sure that we should be doing this. As mentioned in Linear, this ID gets used by the O in trickle paths etc so it is risky to take values from the gateway as-is. We can still add it to the context for logging.

Also not sure why the data is being passed through headers rather than the body with the rest of the request data.